server/models/Notifications.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const notificationsSchema = new mongoose.Schema({
+  notifications: [
+    {
+      isNewNotification: {
```

Comment:
```suggestion
      unread: {
```
This might be a clearer name